lib/nsxv.py

```diff
@@ -614,10 +616,11 @@ def __str__(self):
         term_str = str(term)
         if term_str:
           target.append(term_str)
-      target.append('\n')
 
       # ensure that the header is always first
       target = target_header + target
       target.append('%s' % (_XML_TABLE.get('sectionEnd')))
       target.append('\n')
-    return '\n'.join(target)
+
+      target_as_xml = xml.dom.minidom.parseString(''.join(target))
+    return target_as_xml.toprettyxml(indent=" ")
```

Comment:
I meant to put in 2 spaces here. I think it looks better than one for readability.

However as rob has stated we definitely prefer to match the actual output that NSX provides when you pull the ACL. I have not been able to find out what format I actually get back from NSX, I've gotten different formats based on the tooling I use to query. Do you know what NSX will actually provide back?